pytest_tests/testsuites/acl/test_eacl.py

```diff
@@ -252,436 +252,6 @@ def test_extended_acl_deny_replication(
                 storage_nodes,
             )
 
-    @allure.title("Testcase to validate NeoFS system operations with extended ACL")
```

Comment:
I don't see anything similar for basic ACL, is that the case?

Reply:
> I don't see anything similar for basic ACL, is that the case?

I don't quite understand this comment. How is basic ACL and these changes related?

Reply:
> I don't see anything similar for basic ACL, is that the case?

I don't quite understand this comment. How is basic ACL and these changes related?

Reply:
They're not directly related, the question is do we have similar tests for basic ACL? If we're not --- let's have an issue for it.

Reply:
> They're not directly related, the question is do we have similar tests for basic ACL? If we're not --- let's have an issue for it.

https://github.com/nspcc-dev/neofs-testcases/issues/636

Reply:
#634. So you want to merge it as is? Frankly, I've expected it to be solved in a day. If that takes too long, OK, let's move on with stripped test and a separate issue.

Reply:
> #634. So you want to merge it as is? Frankly, I've expected it to be solved in a day. If that takes too long, OK, let's move on with stripped test and a separate issue.

No, I don't want to merge it as is.
But it seems to me that tests for basic acl are a separate task.

I have a question - should basic acl allow changing the system role?
Wouldn't this lead to the same bugs that we fix by removing this functionality from eacl?

And here I will leave the tests that check negative scenarios with system eacl.

Reply:
That's the beauty of basic ACL, it was always implemented in a way forbidding messing with system bits. They can only be set to what they need to be set according to the spec.

Reply:
> > #634. So you want to merge it as is? Frankly, I've expected it to be solved in a day. If that takes too long, OK, let's move on with stripped test and a separate issue.
> 
> No, I don't want to merge it as is. But it seems to me that tests for basic acl are a separate task.
> 
> I have a question - should basic acl allow changing the system role? Wouldn't this lead to the same bugs that we fix by removing this functionality from eacl?
> 
> And here I will leave the tests that check negative scenarios with system eacl.

Fixed